src/pages/item-tracker/index.js

```diff
@@ -5,25 +5,29 @@ import { useTranslation } from 'react-i18next';
 
 import ItemGrid from '../../components/item-grid';
 import useStateWithLocalStorage from '../../hooks/useStateWithLocalStorage';
-import Traders from '../../data/traders.json';
 
-import quests from '../../data/quests.json';
+//import quests from '../../data/quests.json';
 import { useItemsQuery } from '../../features/items/queries';
+import { useTradersQuery } from '../../features/traders/queries';
+import { useQuestsQuery } from '../../features/quests/queries';
 
 import './index.css';
 
 function ItemTracker() {
+    const { data: quests } = useQuestsQuery();
     const [questData, setQuestData] = useStateWithLocalStorage(
         'quests',
-        quests.data,
+        quests,
     );
+    console.log(questData); 
```

Comment:
🗑️

Reply:
This page is and has been incomplete, but fixed.